APItokenmanager/Web/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+class Scope(models.Model):
+    name = models.CharField(max_length=100, unique=True, default="read")
+    description = models.TextField(default="Provides read access to this scope")
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.name
+
+class JWTUser(models.Model):
+    full_name = models.CharField(max_length=255)
+    email = models.EmailField(unique=True)
+    password = models.CharField(max_length=255)
```

Comment:
Any reason this is stored as plain-text?

Take a peek at `django.contrib.auth.hashers.make_password` https://docs.djangoproject.com/en/5.0/topics/auth/passwords/

Reply:
The user registration is done via FastAPI and the password are stored in a hash format. Since, the password is already hashed, I didn't want to re-hash again.

Reply:
Perhaps a higher-level question then -- why is the registration and the token management done within separate services then? 

Also, would you have a link to code where you explicitly hash the password in your FastAPI instance? I'm not seeing that in https://github.com/sensein/fastapi-skeleton, nor in this PR